test/e2e/predictor/test_huggingface.py

```diff
@@ -282,3 +282,61 @@ def test_huggingface_openai_text_2_text():
     assert res["choices"][0]["text"] == "Das ist für Deutschland"
 
     kserve_client.delete(service_name, KSERVE_TEST_NAMESPACE)
+
+
+@pytest.mark.local
+# @pytest.mark.skip("Upload model to gcs")
```

Comment:
Is this still needed?

Reply:
It is skipped for now as the model is not uploaded to gcs yet.